model/countries/IN/IN-formatting-rules.yaml

```diff
@@ -0,0 +1,35 @@
+formatting-rules:
+  address:
+  - street-address-alternative-1
+  - separator: "\n"
+  - postal-code
+  - separator: " "
+  - locality1
+  - separator: "\n"
+  - admin-area1
+  - separator: "\n"
+  - country-name
+  - skip: country  # redundant with country-name
+  - skip: street-address  # redundant with street-address-alternative-1
+  - skip: admin-area1
```

Comment:
Why do you list "skip: admin-area1" here and include it in line 9?

Reply:
Was just WIP. PTAL now.